packages/react/src/components/navigation/Navigation.stories.tsx

```diff
@@ -145,7 +146,7 @@ export const Inline = ({ searchLabel, searchPlaceholder, authenticated, userName
 export const CustomTheme = ({ searchLabel, searchPlaceholder, authenticated, userName, ...args }) => {
   return (
     // @ts-ignore
-    <Navigation {...args}>
+    <Navigation {...args} logo={<Logo src={logoFi} aria-hidden="true" />}>
```

Comment:
Should `aria-hidden` be `true` as default or even fixed as always hidden?

Reply:
Dunno, need to ask in daily perhaps... if the logo is wanted somewhere to not being hidden 🤷

Reply:
The logo is usually link to the service front page. So the link with proper aria-label needs to be available, but the image can always be hidden.

Reply:
Agreed in daily to consult accessibility experts.

Reply:
Accessibility expert says it's not to be hidden from the screen-readers and needs an alt-text if it's shown -> Fixed by setting `alt` property mandatory when aria-hidden is falsy (default -> The logo is shown).

Reply:
Great!